react/header.jsx

```diff
@@ -22,6 +25,7 @@ var Header = React.createClass({
       <div id="header">
         <div id="profile-img">
           <ProfileImage type="header" image={this.props.image}/>
+          <input id="buttons" type="file" name="profile-picture" accept="image/*" ref={(ref) => this.imageFile = ref} onChange={(evt) => this.saveImage(evt)}/>
```

Comment:
When I tried use the onChange={this.saveImage} it didn't fire at all. It seems to be specific to REACT.